client/extensions/woocommerce/state/action-types.js

```diff
@@ -1,3 +1,5 @@
+export const WOOCOMMERCE_API_ERROR = 'WOOCOMMERCE_API_ERROR';
```

Comment:
I prefer actions that are verby, similar to what we do (unfortunately not consistently) elsewhere in calypso, e.g. `RECEIVE_INVITES_ERROR` or `ACTION_RECEIVE_SITE_RECOMMENDATIONS_ERROR`

Could we do `WOOCOMMERCE_API_RECEIVE_ERROR` here instead and maybe `WOOCOMMERCE_API_CLEAR_ERROR` on the next line?